server/functions.lua

```diff
@@ -299,3 +299,17 @@ function QBCore.Functions.IsLicenseInUse(license)
     end
     return false
 end
+
+-- Job Check
+
+function QBCore.Functions.DoesJobExist(job, grade)
```

Comment:
Why does this need to be a separate function?

Reply:
You could run it directly inside the command I guess..

Reply:
But you're not using it elsewhere, if you're running _inside_ the function, do just that